src/app/app.cpp

```diff
@@ -491,6 +620,15 @@ namespace app
         prev_kv->set_lease(value.lease);
       }
 
+      CCF_APP_DEBUG("building custom claims");
+      lskvserverpb::ReceiptClaims claims;
+      auto payload_ptr = std::make_unique<etcdserverpb::PutRequest>(payload);
+      claims.set_allocated_request_put(payload_ptr.release());
```

Comment:
This doesn't seem very pleasant to me having to make a smart pointer just to ignore it but it works.

@jumaffre if you have a nicer way then I'm happy to update it.

Reply:
I've changed to `mutable_*` as it doesn't capture the pointer so doesn't try to free it and I think its a nicer way.